.github/workflows/pr-preview.yml

```diff
@@ -75,17 +79,28 @@ jobs:
           if [ "${BRANCH}" = "release/demo" ]
           then
             aws s3 cp "${WHEELFILE}" s3://core-previews/${PREVIEW_BRANCH}/streamlit-11.11.11-py2.py3-none-any.whl --acl public-read
-            echo -e "Wheel file download link: https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/streamlit-11.11.11-py2.py3-none-any.whl"
+            S3_URL="https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/streamlit-11.11.11-py2.py3-none-any.whl"
           else
             aws s3 cp "${WHEELFILE}" s3://core-previews/${PREVIEW_BRANCH}/ --acl public-read
-            echo -e "Wheel file download link: https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}"
+            S3_URL="https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}"
           fi
+          echo -e "Wheel file download link: ${WHEEL_URL}"
 
           cd ../..
           # env variables don't carry over between gh action jobs
           echo "enable-setup=${{ env.AWS_ACCESS_KEY_ID != '' }}" >> $GITHUB_OUTPUT
           echo "preview-branch=$PREVIEW_BRANCH" >> $GITHUB_OUTPUT
-          echo "s3-url=https://core-previews.s3-us-west-2.amazonaws.com/${PREVIEW_BRANCH}/${WHEELFILE}" >> $GITHUB_OUTPUT
+          echo "s3-url=${S3_URL}" >> $GITHUB_OUTPUT
+
+      - name: Set success deployment status
+        if: success() && github.repository == 'streamlit/streamlit' && github.event_name == 'pull_request' && github.event.sender.type == 'User'
+        uses: pbrandone/create-status-action@b1112f45e23466b401db7d9fc1aa349fb84eacfc
+        with:
```

Comment:
Just a note - we'll have to keep an eye on an update to this action to node 16, with planned deprecation of node 12 actions around this summer (blog [link](https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/)).
<img width="1052" alt="Screen Shot 2023-01-11 at 4 07 42 PM" src="https://user-images.githubusercontent.com/63436329/211944823-5aba86e2-d5fb-498b-990a-86634c07b4d2.png">

Reply:
Good point. I changed it and we use the `actions/github-script@v6` action now.